packages/core-utils/package.json

```diff
@@ -17,8 +17,8 @@
     "lint:check": "eslint .",
     "lint:fix": "yarn lint:check --fix",
     "pre-commit": "lint-staged",
-    "test": "ts-mocha test/**/*.spec.ts",
```

Comment:
What is this diff for?

Reply:
This is noted in the commit message/pr message, but the alias test wasn't running because this glob didn't target it, so now the new glob targets all tests since they are in a flat file structure